src/index.html

```diff
@@ -148,7 +148,7 @@
 
 <div class="container">
     <div class="row justify-content-center">
-        <h1>Pokéclicker</h1>
+        <h1 id="gameTitle">Pokéclicker</h1>
```

Comment:
Can you add a comment here explaining that the money is showed here. Good chance we will remove this title in the future.

Reply:
sure